integration/messaging/test/static/valid-vapid-key-modern-sw/sw.js

```diff
@@ -15,16 +15,23 @@
  * limitations under the License.
  */
 
-import { MessagePayload } from './message-payload';
+importScripts('../constants.js');
+importScripts('../helpers.js');
 
-export enum MessageType {
-  PUSH_RECEIVED = 'push-received',
-  NOTIFICATION_CLICKED = 'notification-clicked'
-}
+// HEAD targets served through express
+importScripts('/firebase-app.js');
+importScripts('/firebase-messaging.js');
 
-export interface InternalMessage {
-  firebaseMessaging: {
-    type: MessageType;
-    payload: MessagePayload;
-  };
-}
+firebase.initializeApp(FIREBASE_CONFIG);
+const messaging = firebase.messaging();
+
+messaging.onBackgroundMessage(payload => {
+  console.log(
+    TAG +
+      'a background message is received in the onBackgroundMessage hook: ' +
```

Comment:
Should there be a space after `TAG` or before `a background ...`?

Reply:
I have the TAG be set as "const TAG = 'FCM_INTEGRATION_TEST: " with an trailing space so  all reference to this constant should be correctly formatted